app/cafe.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    """Represents a cafe with a name attribute."""
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        # Check for vaccination status
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        # Check for vaccine expiration date
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
```

Comment:
The comparison `expiration_date < datetime.date.today()` assumes that `expiration_date` is a `datetime.date` object. Ensure that `expiration_date` is indeed a `datetime.date` object before this comparison. If `expiration_date` is a string, you need to convert it to a `datetime.date` object first.